packages/adyen-integration/src/adyenv3/AdyenV3Form.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FunctionComponent } from 'react';
+
+import { HostedWidgetPaymentComponent } from '@bigcommerce/checkout/hosted-widget-integration';
+import { usePaymentFormContext } from '@bigcommerce/checkout/payment-integration-api';
+import { Modal } from '@bigcommerce/checkout/ui';
+
+const AdyenV3Form: FunctionComponent<any> = ({
+    method,
+    containerId,
+    initializePayment,
+    isAccountInstrument,
+    shouldHideInstrumentExpiryDate,
+    validateInstrument,
+    language,
+    showAdditionalActionContent,
+    cancelAdditionalActionModalFlow,
+    additionalActionContainerId,
+    isPaymentDataRequired,
+    checkoutService,
+    checkoutState,
+    ...rest
+}) => {
+    const { paymentForm } = usePaymentFormContext();
+    const customer = checkoutState.data.getCustomer();
+    const instruments = checkoutState.data.getInstruments(method) || [];
+    const isInstrumentFeatureAvailable =
+        !customer?.isGuest && Boolean(method.config.isVaultingEnabled);
+
+    return (
+        <>
+            <HostedWidgetPaymentComponent
+                {...rest}
+                containerId={containerId}
+                deinitializePayment={checkoutService.deinitializePayment}
```

Comment:
As far as I see, `deinitializePayment` method was provided to props:
https://github.com/bigcommerce/checkout-js/pull/1433/files#diff-bd5fbd171f4dbccef385dad48890f89a0dd218f9db63bc0248657f20bac99789R184